src/sentry/api/invite_helper.py

```diff
@@ -38,17 +39,46 @@ def get_invite_cookie(request):
 
 
 class ApiInviteHelper(object):
-    def __init__(self, instance, request, member_id, token, logger=None):
+    @classmethod
+    def fromCookieOrEmail(cls, request, organization, email, instance=None, logger=None):
```

Comment:
Should this be `from_cookie_or_email`?

Reply:
Oh strange. Not sure why this style felt right for me for a classmethod